Assets/_BForBoss/_Weapons/Scripts/Bullets/BulletSpawner.cs

```diff
@@ -8,6 +8,8 @@ public class BulletSpawner : MonoBehaviour
         [SerializeField] private BulletBehaviour[] _bulletPrefabs;
         private ObjectPooler<BulletBehaviour>[] _pools = null;
 
+        [SerializeField, Layer] private int _layer = 8; //default is Bullet layer
```

Comment:
Would it be more accurate to find the layer via string? since sorting and changing the layers would affect this

Reply:
Yeah I was on the fence about this. The downside of having a string is that it plays less well with the inspector, with an int its just a matter of selecting the layer in a dropdown. With a string, you're right its a little more robust to later changes to the layer order, but initially setting it is more of a process, and you have to handle converting to an int at runtime anyway